src/type-layout.md

```diff
@@ -86,9 +86,9 @@ String slices are a UTF-8 representation of characters that have the same layout
 
 ## Tuple Layout
 
-Tuples do not have any guarantees about their layout.
+Tuples are laid out according to the default representation.
```

Comment:
What does "the default representation" refer to here?

Reply:
The section with most of my changes has the heading "The Default Representation", and that's what I'm referring to. I don't know if there's a better way to phrase it, or perhaps add a link to the heading?

Reply:
+1 a link to the heading would be great, and with an explicit note that they may not be the same representation as a struct with equivalent fields.